_assets/js/main.js

```diff
@@ -875,6 +875,10 @@ jQuery(document).ready(function($) {
             if (e.key == 'Escape' && $panel.is(':visible')) {
               closePanel($link, $panel);
             }
+          })
+          .on('scroll', function() {
```

Comment:
This event should be namespaced - see above for examples. It should also not operate if there's no panel open.

Reply:
See my latest comment, I think scroll events might not be the best hook for this.